css-positioning/relative.css

```diff
@@ -0,0 +1,46 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  right: 320px;
```

Comment:
I told you when introducing the exercise that you don't need to worry about moving these left or right at all 🤔